src/officehours_api/nested_serializers.py

```diff
@@ -1,12 +1,20 @@
 from rest_framework import serializers
 from django.contrib.auth.models import User
-from officehours_api.models import Meeting, Attendee
+from officehours_api.models import Meeting, Attendee, Profile
 
 
 class NestedUserSerializer(serializers.ModelSerializer):
+    phone_number = serializers.CharField(source='profile.phone_number')
     class Meta:
         model = User
-        fields = ['id', 'username', 'first_name', 'last_name']
+        fields = ['id', 'username', 'first_name', 'last_name', 'phone_number']
+
+""" class NestedProfileSerializer(serializers.ModelSerializer):
+    user = NestedUserSerializer(required=True)
+
+    class Meta:
+        model = Profile
+        fields = ['user', 'phone_number'] """
```

Comment:
rm (you can always find deleted code in your git history later. I see what you were thinking here, and I'm not sure yet which is the best way)